microcloud/service/microovn.go

```diff
@@ -54,7 +55,30 @@ func (s OVNService) Client() (*client.Client, error) {
 
 // Bootstrap bootstraps the MicroOVN daemon on the default port.
 func (s OVNService) Bootstrap() error {
-	return s.m.NewCluster(s.name, util.CanonicalNetworkAddress(s.address, s.port), 2*time.Minute)
+	err := s.m.NewCluster(s.name, util.CanonicalNetworkAddress(s.address, s.port), 2*time.Minute)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 1 * time.Minute)
```

Comment:
Is there a global CTX that can be used in case init process is canceled BTW?

Reply:
Ah good point, these aren't in the `main` package so they shouldn't use `Background()`